src/components/VisuallyHidden.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import styled from "styled-components";
+
+const HiddenStyled = styled.div`
+  display: inline-block;
+  position: absolute;
+  overflow: hidden;
+  clip: rect(0 0 0 0);
+  height: 1em;
+  width: 1em;
+  margin: -1em;
+`;
+
+const VisuallyHidden = ({ children }) => {
+  return <HiddenStyled>{children}</HiddenStyled>;
+};
```

Comment:
You probably don't need to create a React component here. It can just be a styled-component like:

```js
export const VisuallyHidden = styled.div`
  display: inline-block;
  position: absolute;
  overflow: hidden;
  clip: rect(0 0 0 0);
  height: 1em;
  width: 1em;
  margin: -1em;
`;
```

Children will be put inside it automatically.